website/src/content/api-reference/custom-webpack-config.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: "Custom Webpack Config"
+---
+
+<Subtitle>
+The utilities allows you create a custom webpack configuration for a Merchant Center Application.
+</Subtitle> 
```

Comment:
When using JSX elements in MDX (at least in the current version), remember to keep empty lines between the JSX element and the content, otherwise the content won't be formatted.

```
<Subtitle>

My content

</Subtitle> 
```

However, I'm not sure we actually need this `Subtitle` section. I think we can omit it.